app/tests/spec/views/change_password.js

```diff
@@ -222,6 +224,7 @@ function (chai, _, $, sinon, AuthErrors, FxaClient, Metrics, p,
               assert.isTrue(view.fxaClient.signIn.calledWith(
                       EMAIL, newPassword, relier));
               assert.isTrue(user.setSignedInAccount.calledWith(account));
+              assert.isTrue(broker.afterChangePassword.called);
```

Comment:
Perhaps use `calledWith(account)` to make sure it's called with the expected argument.